networkx/algorithms/flow/__init__.py

```diff
@@ -1,12 +1,6 @@
-from .maxflow import *
-from .mincost import *
-from .edmondskarp import *
-from .preflowpush import *
-from .shortestaugmentingpath import *
-from .capacityscaling import *
-from .networksimplex import *
-from .utils import build_flow_dict, build_residual_network
-
+# this import is used to combine the __all__ varibles.
```

Comment:
I understand how `__all__` works, but you don't need to specify it if it lists everything in your module.